Adds options to which models are included

cmcau opened the pull request:
Pull Request
**Are you a current Fivetran customer?** 
Partner
Chris McClellan
Director
Visualisedata Pty Ltd
AUSTRALIA

**What change(s) does this PR introduce?** 
Adds options to dbt_project.yml to optionally load some of the models.  I wanted to use this package to show what Fivetran was doing (during the trial period), but some of the models are not available and that causes an error.  With these changes I don't get any errors and I can report on the data because the run finishes without any errors

**Did you update the CHANGELOG?** 
- [x] Yes

**Does this PR introduce a breaking change?**
<!--- Does this PR introduce changes that will cause current package users' jobs to fail or require a `--full-refresh`? -->
<!--- To select a checkbox you simply need to add an "x" with no spaces between the brackets (eg. [x] Yes). -->
- [x] Yes (please provide breaking change details below.)
- [ ] No  (please provide an explanation as to how the change is non-breaking below.)
I'm going to say YES to be safe.  I only thought of that now - if users don't have the vars defined then there will be an error I guess.  I haven't tested this, but I'm happy to do the work if this is an issue that needs to be fixed and this PR is going to benefit the package.

**Did you update the dbt_project.yml files with the version upgrade (please leverage standard semantic versioning)? (In both your main project and integration_tests)** 
<!--- The dbt_project.yml and the integration_tests/dbt_project.yml files contain the version number. Be sure to upgrade it accordingly -->
<!--- To select a checkbox you simply need to add an "x" with no spaces between the brackets (eg. [x] Yes). -->
- [x] Yes

**Is this PR in response to a previously created Bug or Feature Request**
<!--- If an Issue was created it is helpful to track the progress by linking it in the PR. -->
<!--- To select a checkbox you simply need to add an "x" with no spaces between the brackets (eg. [x] Yes). -->
- [ ] Yes, Issue/Feature [link bug/feature number here]
- [x] No 

**How did you test the PR changes?** 
<!--- Proof of testing is required in order for the PR to be approved. -->
<!--- To check a box, remove the space and insert an x in the box (eg. [x] CircleCi). --> 
<!--- To select a checkbox you simply need to add an "x" with no spaces between the brackets (eg. [x] Yes). -->
- [ ] CircleCi <!--- CircleCi testing is only applicable to Fivetran employees. --> 
- [x] Local (please provide additional testing details below)

I tested it with all the vars properly defined in the root dbt_project.yml and it passed all tests.
I DID NOT test when the vars are not defined in the root dbt_project.yml

**Select which warehouse(s) were used to test the PR**
<!--- To check a warehouse remove the space and insert an x in the box (eg. [x] Bigquery). --> 
<!--- To select a checkbox you simply need to add an "x" with no spaces between the brackets (eg. [x] Yes). -->
- [ ] BigQuery
- [ ] Redshift
- [x] Snowflake
- [ ] Postgres
- [ ] Databricks
- [ ] Other (provide details below)

**Provide an emoji that best describes your current mood**
<!--- For a complete list of markdown compatible emojis check our this git repo (https://gist.github.com/rxaviers/7360908)  --> 
:grinning: hopefully, but a lot of :worried: at the same time

**Feedback**

We are so excited you decided to contribute to the Fivetran community dbt package! We continue to work to improve the packages and would greatly appreciate your [feedback](https://www.surveymonkey.com/r/DQ7K7WW) on our existing dbt packages or what you'd like to see next.

fivetran-joemarkiewicz commented:
Hey @cmcau thanks so much for opening this PR!! 

These changes make complete sense and I understand why you would want to remove these models from the run as they do not exist during the trial phase. My one question I have is regarding the value of the end models with these disabled. Are you still able to gain actionable insights in your opinion with these disabled?

cmcau commented:
Well, I don't know what I'm missing right ? ;) 
I have created a visualisation that shows the data in the log_audit table (so I can see Deletes, Updated and Replaced or Inserted by Connector and Table Name.  

Active Volume seems to be the most interesting one that I'm missing (but I'm only going on the name of the model)
